src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,39 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int STARTING_INT = -1;
+    private static final int MAX_TAB_SIZE = 12;
+    private static final int FULL_INT = 11;
```

Comment:
This constant is related to MAX_TAB_SIZE . Changing only one leads to an error (inconsistency).

INDEX instead of INT would be better.